### PR TITLE
feat: NODE_ENV is configured in the frontend

### PR DESCRIPTION
**Pull request by palash-signoz**
Close https://github.com/SigNoz/engineering-pod/issues/362

### CODE REVIEW

File: frontend/src/constants/env.ts [hunk 1]
```diff
@@ -3,4 +3,6 @@ export const ENVIRONMENT = {
 		process?.env?.FRONTEND_API_ENDPOINT ||
 		process?.env?.GITPOD_WORKSPACE_URL?.replace('://', '://8080-') ||
 		'',
+	isProduction: process?.env?.NODE_ENV === 'production',
```
Comment (pranshuchittora) [inline frontend/src/constants/env.ts:6]: Why `isProduction` required are we using this anywhere.
I can see we already have `NODE_ENV`

Comment (palash-signoz) [reply frontend/src/constants/env.ts:6]: nopes not as of now... _(with Axolo)_

Comment (pranshuchittora) [reply frontend/src/constants/env.ts:6]: Oh in that case let’s remove it, let;s add it when there’ll be the requirement _(with Axolo)_

Comment (palash-signoz) [reply frontend/src/constants/env.ts:6]: ok will remove this one _(with Axolo)_

Comment (pranshuchittora) [reply frontend/src/constants/env.ts:6]: Pls update this

Comment (palash-signoz) [reply frontend/src/constants/env.ts:6]: 1 min _(with Axolo)_

Comment (palash-signoz) [reply frontend/src/constants/env.ts:6]: done @pranshuchittora _(with Axolo)_

**Review by pranshuchittora — APPROVED** [review]
Let's get this reviewed from Ankit as well because this includes build config changes

**Pull request merged**